git-coverage

```diff
@@ -792,8 +809,13 @@ def main(argv):
             patches_by_filename[filename] = []
         patches_by_filename[filename].append(patch)
 
+    omit_patterns = get_omit_patterns()
     # Now go through and calculate coverage
     for (filename, patches) in patches_by_filename.items():
+        if any([fnmatch.fnmatch(filename, omit_pattern)
+            for omit_pattern in omit_patterns]):
+            continue
```

Comment:
I also requested a list of the files that were skipped and a list of the files which were found to be fully covered - a print statement here will easily show the files that are skipped, but you'll need to add logic after line 812 to print when a file was fully covered